### PR TITLE
Add Meicoo

### PR DESCRIPTION
**Pull request by ghost**
Code for boost swap provided by heart swap

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -1367,6 +1367,46 @@ let BattleMovedex = {
...
+				// @ts-ignore
+				sourceBoosts[i] = source.boosts[i];
+			}
+
+			target.setBoost(sourceBoosts);
+			source.setBoost(targetBoosts);
+
+			this.add(`c|%Meicoo|cool quiz`);
+
+			this.add('-swapboost', source, target, '[from] move: Heart Swap'); // Gives a message notifying the swapping of boosts
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: `'[from] move: /scavenges u'`

**Review by HoeenCoder — COMMENTED** [review]
You didnt modify the mega forme to give it No Guard

**Comment by ghost** [comment]
screenshot 74
erm

**Comment by HoeenCoder** [comment]
ah

**Pull request merged**